src/views/Todos.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import AddTask from '../components/AddTask';
+import TodoItem from '../components/TodoItem';
+import { addNewTodo, fetchTodos } from '../services/todos';
+import './Todos.css';
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [newTask, setNewTask] = useState('');
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const resp = await fetchTodos();
+      setTodos(resp);
+    };
+    fetchData();
+  }, []);
+
+  const handleNewTask = async (e) => {
+    e.preventDefault();
+    try {
+      await addNewTodo(newTask);
+      const resp = await fetchTodos();
+      setTodos(resp);
```

Comment:
We can also prevent another network request call here by using prevState to set our old todos and pass in the new one.